tests/installation/test_authenticator.py

```diff
@@ -1,3 +1,4 @@
+import pathlib
```

Comment:
```suggestion
from pathlib import Path
```

Reply:
There's a few reasons to prefer `import pathlib` to `from pathlib import Path`. The primary one is that it follows the import pattern used in this module already with requests, pytest, and other. Secondly, `pathlib.Path` is more clear and explicit the type of `Path` object being used since the name Path is common and can be overloaded. The last reason, though it isn't really relevant here, is that using the `import path` style makes it much easier to mock internal functions in pathlib if necessary.

Reply:
I would say those reasons are rather subjective and not necessarily universal. Further, I do not think we use anything other than `pathlib.Path` as `Path` through the code base. I recommended the change as this is how `Path` is import through the rest of the code. As for mocking, I suspect we will have to patch anyway within `poetry.*` in most cases.

Reply:
OK. Will update

Reply:
Since 1.2.0 looks imminent I thought I'd bump this thread- this is the last cert-related issue that is blocking my company from being able to use poetry generally for our internal packaging, and since I've already converted a number of our packages (and have been using a private fork with a couple of cert fixes) I'm desperate to get this change merged. Is there a compromise we could reach here?

Reply:
I will try to get some cycles to work on this today. The latest requests for changes broke a bunch of unit tests, so I will have to figure out what happened there. Sorry it's been slow to merge.

Reply:
Updates pushed. Tests passed locally; will see what the pipeline does when it's greenlit to run

Reply:
The python3.6 tests on FreeBSD failed with a strange (or missing) error: https://github.com/python-poetry/poetry/pull/3490/checks?check_run_id=2657871763

Maybe that test just needs to be re-run and the other workflows could be approved by a maintainer see if there are other issues this PR needs to address? @abn

Reply:
Yes, that failure seems to be unrelated to my change. I will try to act on any more feedback on this quickly, since folks are waiting on this feature.

Reply:
This has been updated as requested.